ionic/install_windows_src.md

```diff
@@ -0,0 +1,201 @@
+   ```bash
+   mkdir gz-ws
+   cd gz-ws
+   mkdir src
+   ```
+
+7. Then clone the repositories
+   ```bash
+   # CMD
+   curl -sk https://raw.githubusercontent.com/gazebo-tooling/gazebodistro/master/collection-ionic.yaml -o collection-ionic
+   vcs import src < collection-ionic
+
+   # PowerShell
+   curl https://raw.githubusercontent.com/gazebo-tooling/gazebodistro/master/collection-ionic.yaml -o collection-ionic
+   vcs import --input collection-ionic src
+   ```
+
+## Building the Gazebo Libraries
+
+Once the compiler and all the sources are in place it is time to compile them.
+Start the procedure by navigating to your workspace and listing the packages
+recognized by `colcon`:
+
+```bash
+colcon graph
+```
+
+`colcon graph` should list the Gazebo libraries with an
+[interdependency diagram](https://colcon.readthedocs.io/en/released/reference/verb/graph.html#example-output).
+If that is the case, then you are ready to build the whole set of libraries:
+
+```bash
+colcon build --cmake-args -DBUILD_TESTING=OFF --merge-install --packages-up-to gz-sim8 gz-tools2
+```
+Tests are turned off as they are not currently supported on Windows.
+
+To build a specific package with all its dependent packages:
+
+```bash
+colcon build --merge-install --packages-up-to PACKAGE_NAME
+```
+
+To build a single package:
+
+```bash
+colcon build --packages-select PACKAGE_NAME
+```
+
+Visit [colcon documentation](https://colcon.readthedocs.io/en/released/#) to view more `colcon` build and test options.
+
+If there are no errors, all the binaries should be ready to use.
+
+## Using the workspace
+
+The workspace needs to be sourced every time a new terminal is used (
+and Conda environment activated before that).
+
+Run the following command to source the workspace:
+
+```bash
+# CMD
+call install\setup.bat
+
+# PowerShell
+.\install\setup.ps1
+```
+
+This is the end of the source install instructions; head back to the [Getting started](/docs/all/getstarted)
+page to start using Gazebo!
+
+> **NOTE**
+> As Gazebo GUI is not yet working, running `gz sim` will not work. You can run only the server with
+> ```cmd
+> gz sim -s -v
+> ```
+
+> **NOTE**
+> If your username contains spaces (which is quite common on Windows), you will probably get errors
+>  saying `Invalid partition name [Computer:My User With Spaces]`. Fix this by changing `GZ_PARTITION`
+>  to something else:
+> ```cmd
+> set GZ_PARTITION=test
+> ```
+> Remember to set the same partition in all other consoles.
+
+### Gazebo GUI workaround
+
+Although running `gz sim` without arguments is not supported on Windows,
+and `gz sim -g` is also not supported, there is a workaround you can apply
+to be able to launch `gz sim -g` on Windows.
+
+> Manually comment [these lines](https://github.com/gazebosim/gz-sim/blob/gz-sim7_7.5.0/src/cmd/cmdsim.rb.in#L497-L501) and [these lines](https://github.com/gazebosim/gz-sim/blob/gz-sim7_7.5.0/src/cmd/cmdsim.rb.in#L558-L562) in file `install\lib\ruby\gz\cmdsim8.rb`.
```

Comment:
```suggestion
> Manually comment [these lines](https://github.com/gazebosim/gz-sim/blob/gz-sim7_7.5.0/src/cmd/cmdsim.rb.in#L497-L501) and [these lines](https://github.com/gazebosim/gz-sim/blob/gz-sim7_7.5.0/src/cmd/cmdsim.rb.in#L558-L562) in file `install\lib\ruby\gz\cmdsim9.rb`.
```

Reply:
https://github.com/gazebosim/docs/pull/405/commits/1d0f9928f8b848adb6ca97aab899bd91bbf4a913 but I noticed that this is using the `gz-sim7-7.5.0` tag. We'll probably need to update it once we create a stable branch.